enumerable.rb

```diff
@@ -0,0 +1,88 @@
+module Enumerable
+  def my_each
+    count = 0
+    while count < self.size
+      yield(self[count])
+      count += 1
+    end
+
+  def my_each_width_index
+    index = 0
+    for element in self
+      yield(self[element], element)
+      index += 1
+  end
+
+  def my_select
+    result= []

```

Comment:
`my_select` doesn't work. It always returns nil.
**Hint** You need to return the `result` array

Reply:
done

Reply:
`my_select` :+1: